metadata-ws/src/main/java/uk/ac/ebi/ampt2d/metadata/persistence/repositories/SampleCustomRepository.java

```diff
@@ -0,0 +1,43 @@
+/*
+ *
+ * Copyright 2019 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package uk.ac.ebi.ampt2d.metadata.persistence.repositories;
+
+import org.springframework.data.querydsl.QueryDslPredicateExecutor;
+import org.springframework.data.repository.NoRepositoryBean;
+import org.springframework.data.repository.PagingAndSortingRepository;
+import uk.ac.ebi.ampt2d.metadata.persistence.entities.QSample;
+import uk.ac.ebi.ampt2d.metadata.persistence.entities.Sample;
+
+@NoRepositoryBean
+public interface SampleCustomRepository extends PagingAndSortingRepository<Sample, Long>,
```

Comment:
```suggestion
public interface SampleRepositoryCustom extends PagingAndSortingRepository<Sample, Long>,
```

Reply:
Thanks, fixed in https://github.com/EBIvariation/metadata-ws/pull/55/commits/7892cdfab1d402f1072d29b0b4cb4dc8720da035